processing/src/main/java/io/druid/segment/filter/ExtractionFilter.java

```diff
@@ -57,12 +55,13 @@ public ExtractionFilter(
     if (allDimVals != null) {
       for (int i = 0; i < allDimVals.size(); i++) {
         String dimVal = allDimVals.get(i);
-        if (value.equals(fn.apply(dimVal))) {
+        if (value.equals(Strings.nullToEmpty(fn.apply(dimVal)))) {
```

Comment:
That's actually not consistent with the other filters.  It's possible to pass `"value": null` to them and it is a little weird to not support that for these as well.